AsyncDisplayKit/Details/ASChangeSetDataController.mm

```diff
@@ -81,8 +81,16 @@ - (void)endUpdatesAnimated:(BOOL)animated completion:(void (^)(BOOL))completion
     
     ASDataControllerLogEvent(self, @"triggeredUpdate: %@", _changeSet);
     
+    // TODO: move
+    [super prepareMoveItemChanges:[_changeSet itemChangesOfType:_ASHierarchyChangeTypeMove]];
+    
+    // Prepare UIKit batch updates
     [super beginUpdates];
     
+    for (_ASHierarchyMoveItemChange *change in [_changeSet itemChangesOfType:_ASHierarchyChangeTypeMove]) {
+      [super moveRowAtIndexPath:change.fromIndexPath toIndexPath:change.toIndexPath withAnimationOptions:change.animationOptions];
+    }
+    
```

Comment:
I agree, they should be processed along with other inserts and deletes. We will, however, have to _not_ tell UIKit about those inserts/deletes and instead tell UIKit about the move. That may require a somewhat radical change in how we inform UIKit about updates, which I'm open to.